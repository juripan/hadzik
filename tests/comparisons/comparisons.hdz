naj x = 10
naj y = 5
naj t = -5

kec (x > y aj y > t) aj x > t{
    hutor('A')
}
kec x == 10 abo y == 10{
    hutor('B')
}

kec ne(x == 15){
    hutor('C')
}