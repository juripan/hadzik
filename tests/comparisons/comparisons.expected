stdout: ABC| stderr: | returncode: 0